docs/user/dashboard/lens.asciidoc

```diff
@@ -22,6 +22,51 @@ image::images/lens_data_info.png[Data summary window with Other]
 
 NOTE: The sum of all the *Other* fields can equal more than 100% by a small amount.
 
+[float]
+[[drag-and-drop-keyboard-navigation]]
+=== Create your visualizations faster with keyboard navigation
+
+Lens has a fully accessible, continuously improved drag and drop system. By using keyboard navigation instead of a mouse you can significantly speed up your work.
```

Comment:
I have a preference for reference-style docs here, because I find the tutorials for drag and drop to be repetitive. The whole point is that the interactions are all very similar, so why can't we provide a general reference?

* Space to start
* Right/left to switch groups
* Up/down to reorder
* Escape to cancel

The other thing that I don't see explained in this tutorial is how to set focus on a field. It's not as easy as it seems for a mouse user who has never done it, and I think we need to actually explain it.

Reply:
Yeah, I was struggling here to decide if it's better to learn by example or through general instructions. I think you're right that interaction is quite basic so maybe general instructions will be fine. Will modify it along with focus instructions! Thanks Wylie!